DesignSafe-CI/portal
# task/DES-2116 - Added new metadata for Google Datasets

It should now recognize the related entities of publications and be cleaner.

## Overview: ##
Google Data Search is indexing only at the Project level. Instead, datasets with dois at the experiment/mission/sim levels are not being indexed. We are mostly interested in marking up our cited publications with DOI.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2116](https://jira.tacc.utexas.edu/browse/DES-2116)

## Summary of Changes: ##
Nested related entities in the publications' JSON+LD for Google Dataset. They should provide all the information that the publication does, plus their dois


## Testing Steps: ##
1. Navigate to any publication with related entities (ie. Experimental publications)
2. View Page Source (CMD+OPTION+U)
3. Find `<script type="application/ld+json">`
4. Copy/paste JSON into the Rich Results test:  https://search.google.com/test/rich-results
5. Make sure that `relatedIdentifier_X` appears (where X is an integer) and the entry generally makes sense.

## UI Photos:

## Notes: ##
The most thorough test for this will be releasing it and seeing how Google Dataset picks up on it.
